app/controllers/hosts_controller.rb

```diff
@@ -521,21 +513,8 @@ def submit_rebuild_config
 
   def submit_multiple_build
     reboot = params[:host][:build] == '1' || false
-
-    missed_hosts = @hosts.select do |host|
-      success = true
-      forward_url_options(host)
-      begin
-        host.built(false) if host.build? && host.token_expired?
-        host.setBuild
-        host.power.reset if host.supports_power_and_running? && reboot
-      rescue => error
-        message = _('Failed to redeploy %s.') % host
-        Foreman::Logging.exception(message, error)
-        success = false
-      end
-      !success
-    end
+    @hosts.each { |host| forward_url_options(host) }
```

Comment:
```suggestion
    @hosts.each(&:forward_url_options)
```

Reply:
Does not work  because the forward_url_options method accepts a host value

```ruby
def bar(x)
   x+100
end  
=> :bar
[3] pry(main)> [1,2,3,4].each(&:bar)
ArgumentError: wrong number of arguments (given 0, expected 1)
```